jscomp/others/js_json.mli

```diff
@@ -236,4 +236,9 @@ Js.log \@\@ Js.Json.stringifyAny [| "foo"; "bar" |]
 *)
 
 
+(** Best-effort serialization, it tries to seralize as 
+  many objects as possible and deserialize it back*) 
 
+val deserializeExn : string -> 'a 
+
+val serializeExn : 'a -> string option 
```

Comment:
why does `serializeExn` return an `string option`?

Reply:
`JSON.stringify(Symbol())` returns undefined, maybe we just raise in this case to simplify things